settings.js

```diff
@@ -211,6 +211,7 @@ window.spinnakerSettings = {
     required: false,
   },
   pubsubProviders: ['google'], // TODO(joonlim): Add amazon once it is confirmed that amazon pub/sub works.
+  plugins: [],
```

Comment:
You'll probably need to add this to the `ISpinnakerSettings` interface for completeness sake.